### ISSUE TITLE
Batch Size = 1?

### ISSUE THREAD

**Issue by kmader** [voxelmorph/voxelmorph#6]
in the paper you state: `each training batch consists of one pair of volumes` which appears to be the case in your code 
https://github.com/voxelmorph/voxelmorph/blob/568af9e215a851777fa2c9337fef94fbf75d8d8d/src/train.py#L73-L75

Is there is reason (GPU memory probably a big one) why you didn't try larger batch-sizes? Intuitively it would seem that large sizes would lead to a more stable gradient descent.

**Comment by adalca** [comment]
Yup, that's the reason. If you're using other data or GPUs where you can afford more volumes per batch, feel free to use more!.
